openraft/src/core/admin.rs

```diff
@@ -184,12 +163,10 @@ impl<'a, D: AppData, R: AppDataResponse, N: RaftNetwork<D>, S: RaftStorage<D, R>
             return Ok(());
         }
 
-        let curr = &self.core.effective_membership.membership;
-
+        let curr = self.core.effective_membership.membership.clone();
+        let diff_members = members.difference(curr.all_members());
         let mut new_config = curr.next_safe(members.clone());
 
-        tracing::debug!(?new_config, "new_config");
-
```

Comment:
Let's keep this debug statement before we are all sure about the correctness of all future modifications to raft.